### ISSUE TITLE
Mutlimodule Dokka v1.4.0-rc

### ISSUE THREAD

**Issue by Jaimss** [Kotlin/dokka#1283]
**Question**
I have a multimodule gradle project and am trying to use the `dokkaHtmlCollector` as the [blog](https://blog.jetbrains.com/kotlin/2020/08/dokka-preview-based-on-kotlin-1-4-0-rc/) mentions.
Here is a build scan: https://scans.gradle.com/s/e4cmx4vgjzfws with the [error included](https://scans.gradle.com/s/e4cmx4vgjzfws/failure#1).

**Screenshots**
N/a

**Installation**
- Operating system: Windows 10
- Build tool: Gradle (6.1.1 I think)
- Dokka version: `1.4.0-rc`

**Additional context**
I am not sure what is causing the issue. [This file](https://github.com/Kotlin/dokka/blob/1.4.0-rc/runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/DokkaCollectorTask.kt) seems to be where my error is occuring if it helps.

My build file:
```gradle
plugins {
    id 'maven-publish'
    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
    id 'com.github.johnrengelman.shadow' version '6.0.0'
    id 'org.jetbrains.dokka' version '1.4.0-rc'
}

allprojects {
    apply plugin: 'maven-publish'
    apply plugin: 'org.jetbrains.kotlin.jvm'
    apply plugin: 'com.github.johnrengelman.shadow'

    shadowJar {
        archiveClassifier.set("")
    }

    repositories {
        mavenCentral()
        jcenter()
        maven { url 'https://jitpack.io' }
        maven { url 'https://papermc.io/repo/repository/maven-public/' }
        maven { url 'https://repo.extendedclip.com/content/repositories/placeholderapi/' }
    }

    dependencies {
        compileOnly "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.72"
    }
}

dokkaHtmlCollector {
    outputDirectory = "$projectDir/docs"
}

def ver = '2.1.0'

group 'dev.jaims'
version ver

subprojects {
    group 'dev.jaims.mcutils'
    version ver

    publishing {
        publishing {
            // Publish to Github Packages
            repositories {
                maven {
                    name = "GitHubPackages"
                    url = uri("https://maven.pkg.github.com/jaimss/mcutils")
                    credentials {
                        username = project.findProperty("github.username") ?: System.getenv("USERNAME")
                        password = project.findProperty("github.token") ?: System.getenv("TOKEN")
                    }
                }
            }
            publications {
                gpr(MavenPublication) {
                    from components.java
                }
            }
        }
    }


}

compileKotlin {
    kotlinOptions.jvmTarget = "1.8"
}
compileTestKotlin {
    kotlinOptions.jvmTarget = "1.8"
}
```

**Comment by sellmair** [comment]
Hey @Jaimss, 
you also need to apply the dokka gradle plugin to all submodules that you want to have documentation generated. We are aware, that the current error message is absolutely not helpful at all. We already fixed this in a PR #1194 ☺️

**Comment by sellmair** [comment]
You can track #1289 to get proper error messages ☺️
